resources/sts/4.12/hypershift/ROSASRESupportPolicy.json

```diff
@@ -0,0 +1,461 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "cloudtrail:DescribeTrails",
+        "cloudtrail:LookupEvents",
+        "cloudwatch:GetMetricData",
+        "cloudwatch:GetMetricStatistics",
+        "cloudwatch:ListMetrics",
+        "ec2-instance-connect:SendSerialConsoleSSHPublicKey",
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeAddresses",
+        "ec2:DescribeAggregateIdFormat",
```

Comment:
This is also in the [existing](https://github.com/openshift/managed-cluster-config/blob/master/resources/sts/4.11/sts_support_permission_policy.json#L24) ROSA support policy, can you investigate it a little?

Reply:
Looks like this has been around since these were ported from docs.openshift.com. I tried to look through the docs repo history to find where they came from, but there's too many commits :(